From a48c605b9f3860ee7abaca1a37add444a1b4001a Mon Sep 17 00:00:00 2001
From: Chris Dimich <chris.dimich@boundarydevices.com>
Date: Tue, 8 Mar 2022 14:45:28 -0800
Subject: [PATCH 1/1] start_isp.sh: fix NR_DEVICE_TREE_BASLER variable

The NR_DEVICE_TREE_BASLER variable has the wrong path.

Signed-off-by: Chris Dimich <chris.dimich@boundarydevices.com>
---
 imx/start_isp.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx/start_isp.sh b/imx/start_isp.sh
index 6daabc59c..8a07acfed 100755
--- a/imx/start_isp.sh
+++ b/imx/start_isp.sh
@@ -7,7 +7,7 @@
 #
 
 RUNTIME_DIR="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
-NR_DEVICE_TREE_BASLER=$(grep basler-camera-vvcam /sys/firmware/devicetree/base/soc@0/*/i2c@*/*/compatible -l | wc -l 2> /dev/null)
+NR_DEVICE_TREE_BASLER=$(grep basler-camera-vvcam `find /sys/firmware/devicetree/base/soc@0/ -name compatible | grep i2c` -l | wc -l 2> /dev/null)
 NR_DEVICE_TREE_OV5640=$(grep ov5640 /sys/firmware/devicetree/base/soc@0/*/i2c@*/*/compatible -l | wc -l 2> /dev/null)
 
 # check if the basler device has been enabled in the device tree
-- 
2.31.1

